source/blog/2018-04-01-the-emberjs-times-issue-44.md

```diff
@@ -0,0 +1,63 @@
+---
+title: The Ember.js Times - Issue No. 44
+author: all the people
+tags: Recent Posts, Newsletter, Ember.js Times
+alias : "blog/2018/04/01/the-emberjs-times-issue-44.html"
```

Comment:
should be 04/26

Reply:
I fixed it here, if you want to merge this as a WIP, whatever workflow is easier for you. https://github.com/emberjs/website/pull/3317

Reply:
sorry for all the confusion, did that here https://github.com/emberjs/website/pull/3319